src/scc.jl

```diff
@@ -518,7 +534,7 @@ function post_trial_func(mcs::SimulationInstance, trialnum::Int, ntimesteps::Int
                 intermediate_ce_scc = sum(df_ce .* total_mds[year_index:last_year_index])
                 intermediate_ce_scc_values[(region=:globe, sector=:total, dr_label=dr.label, prtp=dr.prtp, eta=dr.eta, ew=dr.ew, ew_norm_region=dr.ew_norm_region)][trialnum] = intermediate_ce_scc
 
-                norm_cpc_values_ce[(region=:globe, sector=:total, dr_label=dr.label, prtp=dr.prtp, eta=dr.eta, ew=dr.ew, ew_norm_region=dr.ew_norm_region)][trialnum] = cpc[year_index]
+                norm_cpc_values_ce[(region=:globe, sector=:total, ew=dr.ew, ew_norm_region=dr.ew_norm_region)][trialnum] = cpc[year_index]
```

Comment:
Remove fields from the `NamedTuple` used as the key for `norm_cpc_values_ce` Dictionary, we don't use these anymore as they create redundancy